docs/src/main/asciidoc/writing-extensions.adoc

```diff
@@ -2164,7 +2164,7 @@ The only particular aspect of writing Quarkus extensions in Eclipse is that APT
 Quarkus generates a lot of classes during the build phase and in many cases also transforms existing classes.
 It is often extremely useful to see the generated bytecode and transformed classes during the development of an extension.
 
-If you set the `quarkus.package.fernflower.enabled` property to `true` then Quarkus will download and invoke the https://github.com/JetBrains/intellij-community/tree/master/plugins/java-decompiler/engine[Fernflower decompiler] and dump the result in the `decompiled` directory of the build tool output (`target/decompiled` for Maven for example).
+If you set the `quarkus.package.quiltflower.enabled` property to `true` then Quarkus will download and invoke the https://github.com/JetBrains/intellij-community/tree/master/plugins/java-decompiler/engine[Fernflower decompiler] and dump the result in the `decompiled` directory of the build tool output (`target/decompiled` for Maven for example).
```

Comment:
You should also update the link and the link title a bit further in the sentence. Sorry about the iterations.

Reply:
Good point, sorry for missing it when updating the docs